battle.py

```diff
@@ -74,7 +75,8 @@ def fight(player, item):
         while not turn:
             # if enemy is still alive
             if enemy_health > 0:
-                damage = random.randint(0, enemy_max_damage) - defend
+                damage = random.randint(0, enemy_max_damage) - defend * ( armor_protection * round(random.uniform(0.50, 0.90), 1) )
```

Comment:
What does `random.uniform` do?

Reply:
```
round(random.uniform(0.50, 0.90), 1)
```

It generates a random floating number between `.5` and `.9` that will only have 1 number after the '.'.